model/fv_regional_bc.F90

```diff
@@ -4540,14 +4583,26 @@ subroutine regional_boundary_update(array                       &
 !
             i1_blend=i1-nrows_blend_user
             i2_blend=i1-1
+! Original code avoided overlap, but changed this to blend corners from both boundaries
```

Comment:
Same comment as above.

Reply:
I was thinking that these were worth keeping since the intent of the code was changed. In case somebody wants to try it that way again.

Reply:
Git maintains a history of all changes - if anyone wants or needs to reintroduce the old code, they could still do that via git.  I'd still recommend deleting these comments to maintain clean code.

Reply:
Sure, no problem. I pushed an update to delete the code and just left a comment instead.